deeplake/core/transform/transform_tensor.py

```diff
@@ -120,11 +120,14 @@ def append(self, item):
         if not isinstance(item, (LinkedSample, Tensor)) and item is not None:
             shape = getattr(item, "shape", None)
             if shape is None:
-                try:
-                    item = np.asarray(item)
-                    shape = item.shape
-                except ValueError:
+                if isinstance(item, (int, float)):
                     shape = (1,)
+                else:
+                    try:
+                        item = np.asarray(item)
```

Comment:
need to specify dtype, else this will produce warnings

Reply:
this is the same code as before, just moved into if else, not sure if I see the issue

Reply:
if item is a ragged list without uniform shape, numpy produces warnings saying "dtype = object" must be specified

Reply:
Yup, but this could be included in a different fix? The same line existed before too, the only thing that changed is that we don't convert ints and floats into np arrays